common/locales/en/quests.json

```diff
@@ -48,6 +48,8 @@
     "leaveQuest": "Leave Quest",
     "sureLeave": "Are you sure you want to leave the active quest? All your quest progress will be lost.",
     "questOwner": "Quest Owner",
+    "questTaskDamage": "pending damage to boss",
```

Comment:
thanks, missed that.  will fix it in https://github.com/HabitRPG/habitrpg/pull/7951 also.   Dramatically simplifies the notification code there.